docs/preview/02-Features/02-message-handling/03-event-hubs.md

```diff
@@ -69,7 +69,7 @@ public class Program
         services.AddSecretStore(stores => ...);
 
         // Add Azure EventHubs message pump and use OrdersMessageHandler to process the messages.
-        services.AddEventHubsMessagePump("<my-eventhubs-name>", "Arcus_EventHubs_ConnectionString", "<my-eventhubs-blob-storage-container-name>", "Arcus_EventHubs_Blob_ConnectionString")
+        services.AddEventHubsMessagePumpUsingManagedIdentity("<my-eventhubs-name>", "Arcus_EventHubs_ConnectionString", "<my-eventhubs-blob-storage-container-name>", "Arcus_EventHubs_Blob_ConnectionString")
```

Comment:
Also used the managed identity option as a default here, to guide the recommended way.